docs/rancher/cloud-provider.md

```diff
@@ -63,6 +63,106 @@ Starting with Rancher v2.9.0, you can configure a specific folder for cloud conf
 
   ![](/img/v1.4/rancher/rke2-cloud-provider-custom-data-dir.png)
 
+### Manually Deploying to the RKE2 Cluster
+
+1. Generate cloud config data using the script `generate_addon.sh`, and then place the data on every custom node (directory: `/etc/kubernetes/cloud-config`).
+
+  ```bash
+    curl -sfL https://raw.githubusercontent.com/harvester/cloud-provider-harvester/master/deploy/generate_addon.sh | bash -s <serviceaccount name> <namespace>
+  ```
+
+  :::note
+
+    The script depends on `kubectl` and `jq` when operating the Harvester cluster, and functions only when given access to the `Harvester Cluster` kubeconfig file.
+
+    You can find the `kubeconfig` file in one of the Harvester management nodes in the `/etc/rancher/rke2/rke2.yaml` path. The server IP must be replaced with the VIP address.
+
+  Example of content:
+
+    ```yaml
+    apiVersion: v1
+    clusters:
+    - cluster:
+        certificate-authority-data: <redacted>
+        server: https://127.0.0.1:6443
+      name: default
+    # ...
+    ```
+
+    You must specify the namespace in which the guest cluster will be created.
+
+  :::
+
+    Example of output:
+
+    ```yaml
+    ########## cloud config ############
+    apiVersion: v1
+    clusters:
+    - cluster:
+        certificate-authority-data: <CACERT>
+        server: https://HARVESTER-ENDPOINT/k8s/clusters/local
+      name: local
+    contexts:
+    - context:
+        cluster: local
+        namespace: default
+        user: harvester-cloud-provider-default-local
+      name: harvester-cloud-provider-default-local
+    current-context: harvester-cloud-provider-default-local
+    kind: Config
+    preferences: {}
+    users:
+    - name: harvester-cloud-provider-default-local
+      user:
+        token: <TOKEN>
+
+    ########## cloud-init user data ############
+    write_files:
+    - encoding: b64
+      content: <CONTENT>
+      owner: root:root
+      path: /etc/kubernetes/cloud-config
+      permissions: '0644'
+    ```
+
+1. On the Harvester UI, go to the **Cluster Configuration** screen and set the value of **Cloud Provider** to **External**.
```

Comment:
LGTM. 
Just a small suggestion whether we can use a more specific term for the `Harvester UI`, for example `Rancher UI` or `RKE2 cluster creation page`.

Reply:
Hi @TachunLin, yeah, it's clearer for users. Thanks.

Reply:
Thanks @FrankYang0529 for the update :+1: